vhdbuilder/packer/cleanup.sh

```diff
@@ -6,6 +6,14 @@ EXPIRATION_IN_HOURS=168
 # deadline = the "date +%s" representation of the oldest age we're willing to keep
 (( deadline=$(date +%s)-${expirationInSecs%.*} ))
 
+# we will clean old sig image-definitions/versions/galleries, RGs only if 
+# we have overriden the pipeline variable SKIP_IMAGE_DELETION_BACKFILL to False
```

Comment:
Could we run the cleanup as a separate parallel task to avoid running them in all tasks and increase the duration when building VHDs for production?